thesis/chapters/4_thermodynamics.tex

```diff
@@ -0,0 +1,60 @@
+\section{Course Overview}
```

Comment:
Overall, the structure is sound.  I feel like I "miss" the problem and what your adding with just the small summary and then reference to the repository.  I suspect there is a happy medium between this short summary and a full repetition of the repository materials.  For example, you could provide the full problem statement with snippets of the solution code (etc.).  I know we talked about this "balance" but only now that you've got some of the core in place can we really feel out what's the right mix.

Reply:
Changes were made to the Thermodynamics chapter by adding the question statement and noteworthy portions of the solution. A "List of Assignments" was also added to the preamble of thesis